tests/unit/test.index.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+/* global -Promise */
+var Promise = require('es6-promise').Promise;
+var mockery = require('mockery');
+var nodemock = require('nodemock');
+
+module.exports = {
+
+  'findDevices() should work': function(test) {
```

Comment:
Oh, derp. That's from way earlier when the test body was just `test.done()` :dizzy_face: